frontend/lib/evictionfree/declaration-builder/redirect-to-homepage-with-message.tsx

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import { Redirect } from "react-router-dom";
+import { MESSAGE_QS } from "../homepage";
+import { EvictionFreeRoutes } from "../route-info";
+
+export const EvictionFreeRedirectToHomepageWithMessage: React.FC<{}> = () => (
+  <Redirect to={`${EvictionFreeRoutes.locale.home}?${MESSAGE_QS}`} />
```

Comment:
Code climate is complaining about this lack of a test but I did some research and didn't find any super meaningful ways to test just a simple redirect component like this...